CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (<https://github.com/openvinotoolkit/datumaro/pull/1145>)
 
 ### Bug fixes
+- Fix flaky test by setting the join timeout to infinite
+  (<https://github.com/openvinotoolkit/datumaro/pull/1149>)
```

Comment:
I think this is a quick-fix for https://github.com/openvinotoolkit/datumaro/pull/1145, so that we can append this to L41 as follows.

```
- Add multi-threading option to ModelTransform and SAMBboxToInstanceMask
  (<https://github.com/openvinotoolkit/datumaro/pull/1145>, <https://github.com/openvinotoolkit/datumaro/pull/1149>)
```

Reply:
updated